apps/showcase/testing/setup-jest.ts

```diff
@@ -1 +1,30 @@
 import 'jest-preset-angular/setup-jest';
+
+jest.mock('@webcontainer/api',
```

Comment:
Maybe more readable to move the both mocks on external `testing/mocks/webcontainer-api.ts` and `testing/mocks/xterm-xtern.ts` that you can import in this file?

Reply:
I was thinking:

```typescript
// in webcontainer-api.mock.ts

class WebContainerApiMock {
  public static boot = jest.fn(() => Promise.resolve({
    on: jest.fn(() => jest.fn())
  }));
}
jest.mock('@webcontainer/api',
  () => ({
    WebContainer: WebContainerApiMock
  }), {
    virtual: true
  }
);
```

and in this file just:
```typescript
import './mocks/webcontainer-api.mock';
```